### DISCUSSION TITLE
OSX experimental OW binaries for Intel and ARM

### DISCUSSION THREAD

**Discussion by jmalak** [open-watcom/open-watcom-v2#1258] [Announcement]
Now OSX build for 64-bit Intel and ARM is regularly build as other platforms.
It is only usable for cross-compilation from OSX to OW supported platform, because OW doesn't support OSX targets.
These OSX binaries are now included into CI-build snapshot in subdirectories bino64 (Intel version) and armo64 (ARM version).
[CI-build](https://github.com/open-watcom/open-watcom-v2/releases/tag/Last-CI-build)

**Comment by pchapin** [comment]
This is neat. I might try doing this build on my Mac M1-based system. Is there anything I should know before making such an attempt? If the answer is "go read the Wiki," I can handle that! I guess I need to use Clang to bootstrap. What about DOSbox? Are those components just skipped?

**Reply by jmalak** [reply]
Petr, it is standard OW build, only need configure your copy of setvars.sh to your bootstrap compiler and configure DOSBOX-X.
You need to install DOSBOX-X for Mac, because DOSBOX is deprecated. All is configured by bootstrap compiler architecture macro which setup wmake predefined architecture to `__OSX__` and `__OSX_X64__` or `__OSX_ARM64__` macros and this control OW build. The build is enabled for OSX 64-bit architectures Intel and ARM only. OSX binaries are in `rel/bino64` (for intel) or in `rel/armo64` (for ARM64). it builds standard 32-bit OW full build + appropriate OSX 64-bit binaries.

**Comment by winspool** [comment]
Nice step forward.
Thanks.


Normally, i use the snapshot from the latest CI build,
(currently 138MB from 23.3.2024)

but the snapshot from the latest github release build
is bigger (143MB) and more recent (26.3.2024).
